app/modules/topics/components/index.js

```diff
@@ -3,7 +3,9 @@
 import List from './List';
 import Preview from './Preview';
 
-export const components = {
-  list: List,
-  preview: Preview,
+const index = {
+  List,
+  Preview,
 };
+
+export default index;
```

Comment:
Is it best practice to specify the exports separately? If so, maybe put it somewhere in the guidelines.

Reply:
Thats more my personal preference, since it makes it easier to see what exactly is being exported.